proto/observer/blame.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+package zetachain.zetacore.observer;
+
+import "observer/observer.proto";
+
+option go_package = "github.com/zeta-chain/zetacore/x/observer/types";
+
+message Node {
```

Comment:
Can we use the existing NodeAccount structure ? And some fields to it.

That would also allow us to check MsgAddBlame to ensure the nodes being blamed are still part of the network.

Reply:
The node account is defined in crosschain still. We can make this change in the next release, I'll take a note.